### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

This is a housekeeping PR. This PR does not and must not contain changes that affect functionality!

This PR does not block the release of `v10.3.0`. **Feel free to add your own small additions** if you think a section could use a bit of cleanup (nothing big, just small things).

If this is merged before `v10.3.0`, please add the corresponding milestone.

## Type of change

- [x] Improvement (non-breaking change that does improve existing functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/handle_stalled.yml
Comment (polarathene) [inline .github/workflows/handle_stalled.yml:33]: I'm only familiar with MR being used in Gitlab, while PR for Github.

I don't mind either way, but if you change from "pull request" wording to "merge request", it should be consistent. The `closed-pr-message` value still uses "PR".

Likewise the maintainer/assignee mentions are inconsistent with the phrasing for similar messages in this PR, that should be addressed. Might be worth mentioning the ignore label as another option for maintainers too?:

```suggestion
            This merge request has become stale because it has been open for 20 days without activity. This merge request will be closed in 10 days automatically unless:
            - A maintainer removes the label `meta/stale` label or adds the `stale-bot/ignore` label.
            - New activity occurs, such as a new comment.
```

**Review by polarathene — APPROVED** [review]

File: .github/workflows/handle_stalled.yml
Comment (georglauterbach) [reply .github/workflows/handle_stalled.yml:33]: Had been using GitLab before (and find the term "merge request" more fitting). Since GitHub uses pull requests, I will adjust this, right.

**Review by wernerfred — DISMISSED** [review]

File: .github/workflows/handle_stalled.yml
Comment (polarathene) [inline .github/workflows/handle_stalled.yml:1]: Missed changing "Merge Requests" here to "Pull Requests", also not sure if you meant "Stalled" or "Stale". I assume "Handle" is more appropriate than "Handled" for the job name too?

```suggestion
name: "Handle Stale Issues & Pull Requests"
```

**Review by polarathene — APPROVED** [review]

**Review by wernerfred — APPROVED** [review]

**Review by polarathene — APPROVED** [review]

File: target/scripts/startup/setup-stack.sh
Comment (casperklein) [inline target/scripts/startup/setup-stack.sh:1745]: Why? To be consequent, we should then also use, e.g. `/bin/sed` across our scripts.

Comment (georglauterbach) [reply target/scripts/startup/setup-stack.sh:1745]: Because all our scripts use the shebang

```
#! /bin/bash
```

and the user patches scrips should also be executed this way.

Comment (casperklein) [reply target/scripts/startup/setup-stack.sh:1745]: `/bin/bash` is also used, when not providing the full path. You can verify this, with `which bash`.

I find it odd, to prefix the full path to some commands, where not necessary. That's whats the $PATH variable is for 😉

Comment (georglauterbach) [reply target/scripts/startup/setup-stack.sh:1745]: Right, but I could have also used `/usr/bin/env bash`, but this way, I make it explicit and my intentions clear sich program to call how:)

Comment (casperklein) [reply target/scripts/startup/setup-stack.sh:1745]: To be clear: I was not talking about the shebang line. I refered to the quoted script line above.

**Pull request merged**
